spec/models/claim/search_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.describe Claim::Search do
+  subject(:search) { Claim::Search.new(query) }
+
+  let!(:other_claims) { create_list(:claim, 7) }
```

Comment:
Maybe reduce this down a tad, we only really need a single other claim for these specs to give us confidence that non-matching claims aren't returned.

Reply:
This has been marked as resolved but it's still creating seven claims.

Reply:
Arrgh. Sorry!